src/apis/BaseApi.ts

```
@@ -0,0 +1,50 @@
+      ...options,
+    });
+  }
+
+  protected static async delete(path: string, options?: RequestInit) {
+    await fetch(`${BaseApi.baseUrl}${path}`, {
+      method: "DELETE",
+      headers: BaseApi.getHeaders(),
+      ...options,
+    });
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)